### PR TITLE
add Kodi JSON-RPC and EventServer services

### PR DESCRIPTION
**Pull request by nl6720**
https://kodi.wiki/view/JSON-RPC_API#TCP
https://kodi.wiki/view/EventServer#Event_Clients_and_the_EventServer

### CODE REVIEW

File: config/services/kodi-JSON-RPC-TCP.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Kodi JSON-RPC (TCP)</short>
```
Comment (TorontoMedia) [inline config/services/kodi-JSON-RPC-TCP.xml:3]:
```suggestion
  <short>Kodi JSON-RPC</short>
```

File: config/services/kodi-JSON-RPC-TCP.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  <description>A JSON-RPC interface using TCP transport to comunicate with Kodi.</description>
```
Comment (TorontoMedia) [inline config/services/kodi-JSON-RPC-TCP.xml:4]:
```suggestion
  <description>A JSON-RPC interface using TCP transport to communicate with Kodi.</description>
```

File: po/POTFILES.in [hunk 3]
```diff
@@ -110,6 +110,8 @@ config/services/kdeconnect.xml
 config/services/kerberos.xml
 config/services/kibana.xml
 config/services/klogin.xml
+config/services/kodi-EventServer.xml
+config/services/kodi-JSON-RPC-TCP.xml
```
Comment (TorontoMedia) [inline po/POTFILES.in:114]:
```suggestion
config/services/kodi-eventserver.xml
config/services/kodi-rpc.xml
```
Rename files as suggested.

**Review by TorontoMedia — CHANGES_REQUESTED** [review]
Thank you for submitting this pull request. Could you please address my change requests and if possible, add the type semantic to the commit message `feat(service): add Kodi JSON-RPC and EventServer services`

**Review by erig0 — APPROVED** [review]
lgtm. Thanks @nl6720 and @TorontoMedia !

**Comment by marillat** [comment]
Would be nice to know why these files have been added when kodi **already**
 provides these files since kodi 18 released in January 2019 ?
Also kodi install more services thans firewalld
https://github.com/xbmc/xbmc/tree/master/tools/Linux/firewalld-services

Wouldn't it be easier to let applications install firewalld services like systemd does?

**Comment by marillat** [comment]
Thanks

**Comment by erig0** [comment]
> Would be nice to know why these files have been added when kodi **already** provides these files since kodi 18 released in January 2019 ? Also kodi install more services thans firewalld https://github.com/xbmc/xbmc/tree/master/tools/Linux/firewalld-services
> 
> Wouldn't it be easier to let applications install firewalld services like systemd does?

I was unaware Kodi provided its own service definitions.

I have no preference for if they're provided by the package or firewalld. If Kodi prefers to ship them, then great!

I will merge the the revert of this, #1002.

**Pull request merged**